src/controllers/expenseController.js

```diff
@@ -0,0 +1,94 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const filteredExpenses = expensesService.getAllExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
+
+  res.status(200).send(filteredExpenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const expense = expensesService.getExpenseById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+  const user = userService.getUserById(userId);
+
+  if (!user || !spentAt || !title || !amount || !category || !note) {
```

Comment:
It's a good practice to provide more information to the client about what went wrong. Instead of just sending a 400 status code, consider sending a JSON object with a message property that contains a description of the error.